File: rankingEducacional/src/java/model/AlunoFator.java
package model;

public class AlunoFator {

  private String matriculaAluno;
  private String fatorId;
  private boolean resposta;

  public AlunoFator() {}

  public String getMatriculaAluno() {
    return matriculaAluno;
  }

  public void setMatriculaAluno(String matriculaAluno) {
    this.matriculaAluno = matriculaAluno;
  }

  public String getFatorId() {
    return fatorId;
  }

  public void setFatorId(String fatorId) {
    this.fatorId = fatorId;
  }

  public boolean getResposta() {
    return resposta;
  }

  public void setResposta(boolean resposta) {
    this.resposta = resposta;
  }
}

File: App/models/alunoFator.php
<?php

class AlunoFator
{
    private  $matriculaAluno;
    private  $fatorId;
    private  $resposta;

    function __construct()
    {
    }

    public function getMatriculaAluno()
    {
        return $this->matriculaAluno;
    }

    public function setMatriculaAluno($matriculaAluno)
    {
        $this->matriculaAluno = $matriculaAluno;
    }

    public function getFatorId()
    {
        return $this->fatorId;
    }

    public function setFatorId($fatorId)
    {
        $this->fatorId = $fatorId;
    }

    public function getResposta()
    {
        return $this->resposta;
    }

    public function setResposta($resposta)
    {
        $this->resposta = $resposta;
    }
}
